README.markdown

```diff
@@ -139,145 +131,140 @@ class { '::ntp':
 
 ####`package_name`
 
-Determines the name of the package to install.
+Tells Puppet what NTP package to manage. Valid options: string. Default value: 'ntp' (except on AIX and Solaris)
 
 ####`panic`
 
-Determines if ntp should 'panic' in the event of a very large clock skew.
-This defaults to false for virtual machines, as they don't do a great job with keeping time.
+Specifies whether NTP should "panic" in the event of a very large clock skew. Valid options: 'true' or 'false'. Default value: 'true' (except on virtual machines, where major time shifts are normal)
 
 ####`preferred_servers`
 
-List of ntp servers to prefer.  Will append 'prefer' for any server in this list
-that also appears in the servers list.
+Specifies one or more preferred peers. Puppet will append 'prefer' to each matching item in the `servers` array. Valid options: array. Default value: [ ]
 
 ####`restrict`
 
-Sets the restrict options in the ntp configuration.  The lines are
-prefixed with 'restrict', so you just need to list the rest of the restriction.
+Specifies one or more `restrict` options for the NTP configuration. Puppet will prefix each item with 'restrict', so you only need to list the content of the restriction. Valid options: array. Default value for most operating systems:
+
+~~~~
+[
+  'default kod nomodify notrap nopeer noquery',
+  '-6 default kod nomodify notrap nopeer noquery',
+  '127.0.0.1',
+  '-6 ::1',
+]
+~~~~
+
+Default value for AIX systems:
+
+~~~~
+[
+  'default nomodify notrap nopeer noquery',
+  '127.0.0.1',
+]
+~~~~
 
 ####`servers`
 
-Selects the servers to use for ntp peers.
+Specifies one or more servers to be used as NTP peers. Valid options: array. Default value: varies by operating system
 
 ####`service_enable`
 
-Determines if the service should be enabled at boot.
+Tells Puppet whether to enable the NTP service at boot. Valid options: 'true' or 'false'. Default value: 'true'
 
 ####`service_ensure`
 
-Determines if the service should be running or not.
+Tells Puppet whether the NTP service should be running. Valid options: 'running' or 'stopped'. Default value: 'running'
 
 ####`service_manage`
 
-Selects whether Puppet should manage the service.
+Tells Puppet whether to manage the NTP service. Valid options: 'true' or 'false'. Default value: 'true'
 
 ####`service_name`
 
-Selects the name of the ntp service for Puppet to manage.
+Tells Puppet what NTP service to manage. Valid options: string. Default value: varies by operating system
 
 ####`udlc`
 
-Enables configs for undisciplined local clock, regardless of
-status as a virtual machine. 
-
+Specifies whether to enable specialized configuration options for an undisciplined local clock, regardless of its status as a virtual machine. Valid options: 'true' or 'false'. Default value: 'false'
 
 ##Limitations
 
-This module has been built on and tested against Puppet 3.
+This module has been tested on all PE-supported platforms, and no issues have been identified.
```

Comment:
@mhaskel @hunner What platforms is ntp supported on? The list is super long. Is it up-to-date? Do we want to keep it up to date? Is it enough to say that it is for sure supported on the OSes listed in the PE supported compatibility matrix https://forge.puppetlabs.com/supported ?

We're trying to standardize ALL the modules, and this is definitely not what's in the Limitations section for most modules. Should it be? Totally cool if so, I'm just curious.


Operating Systems:

* **RedHat**: 5, 6, 7
* **CentOS**: 5, 6, 7
* **OracleLinux**: 5, 6, 7
* **Scientific**: 5, 6, 7
* **SLES**: 10 SP4, 11 SP1, 12
* **Debian**: 6, 7
* **Ubuntu**: 10.04, 12.04, 14.04
* **Solaris**: 11
* **AIX**: 5.3, 6.1, 7.1


Operating Systems:

* **RedHat**: 5, 6, 7
* **CentOS**: 5, 6, 7
* **OracleLinux**: 5, 6, 7
* **Scientific**: 5, 6, 7
* **SLES**: 10 SP4, 11 SP1, 12
* **Debian**: 6, 7
* **Ubuntu**: 10.04, 12.04, 14.04
* **Solaris**: 11
* **AIX**: 5.3, 6.1, 7.1


Operating Systems:

* **RedHat**: 5, 6, 7
* **CentOS**: 5, 6, 7
* **OracleLinux**: 5, 6, 7
* **Scientific**: 5, 6, 7
* **SLES**: 10 SP4, 11 SP1, 12
* **Debian**: 6, 7
* **Ubuntu**: 10.04, 12.04, 14.04
* **Solaris**: 11
* **AIX**: 5.3, 6.1, 7.1


Operating Systems:

* **RedHat**: 5, 6, 7
* **CentOS**: 5, 6, 7
* **OracleLinux**: 5, 6, 7
* **Scientific**: 5, 6, 7
* **SLES**: 10 SP4, 11 SP1, 12
* **Debian**: 6, 7
* **Ubuntu**: 10.04, 12.04, 14.04
* **Solaris**: 11
* **AIX**: 5.3, 6.1, 7.1


Operating Systems:

* **RedHat**: 5, 6, 7
* **CentOS**: 5, 6, 7
* **OracleLinux**: 5, 6, 7
* **Scientific**: 5, 6, 7
* **SLES**: 10 SP4, 11 SP1, 12
* **Debian**: 6, 7
* **Ubuntu**: 10.04, 12.04, 14.04
* **Solaris**: 11
* **AIX**: 5.3, 6.1, 7.1


Operating Systems:

* **RedHat**: 5, 6, 7
* **CentOS**: 5, 6, 7
* **OracleLinux**: 5, 6, 7
* **Scientific**: 5, 6, 7
* **SLES**: 10 SP4, 11 SP1, 12
* **Debian**: 6, 7
* **Ubuntu**: 10.04, 12.04, 14.04
* **Solaris**: 11
* **AIX**: 5.3, 6.1, 7.1